webrtc.html

```diff
@@ -1670,8 +1670,9 @@ <h4>Set the RTCSessionDescription</h4>
                               <li>
                                 <p><a>Create an RTCRtpTransceiver</a> with
                                 <var>sender</var>, <var>receiver</var> and
-                                <var>direction</var> set to <code>"recvonly"</code>,
-                                and let <var>transceiver</var> be the result.</p>
+                                an <code><a>RTCRtpTransceiverDirection</a></code>
+                                value of <code>"recvonly"</code>, and let
```

Comment:
Hm. This section is part of "apply a description", not of "create transceiver".
Shouldn't the "direction" be the negotiated direction from the SDP?
(it's not really easy to see where this line sits...)

Reply:
The reason it's "recvonly" is that if you receive a "sendrecv" offer but don't have anything to send (no `addTrack`-matched transceiver), you want to reply with a "recvonly" answer by default. So this is correct/intentional.